server/views/pages/professionalContacts/professionalContactsPage.njk

```diff
@@ -56,18 +56,22 @@
                     </dd>
                 </div>
 
-                <div class="govuk-summary-list__row" data-qa="contact-address">
-                    <dt class="govuk-summary-list__key">
-                        Address
-                    </dt>
-                    <dd class="govuk-summary-list__value">
-                        {% if contact.address and not contact.address.noFixedAddress %}
-                            {{ arrayToLines(contact.address | addressToLines, "Not entered") }}
-                        {% else %}
-                            Not entered
-                        {% endif %}
-                    </dd>
-                </div>
+                {% if contact.relationshipDescription !== "Prison Offender Manager"
```

Comment:
Might be better to use `contact.relationship` which is just a code (e.g. `POM`), just in case the text changes in future?